ci/lint/06_script.sh

```diff
@@ -9,7 +9,12 @@ export LC_ALL=C
 GIT_HEAD=$(git rev-parse HEAD)
 if [ -n "$CIRRUS_PR" ]; then
   COMMIT_RANGE="${CIRRUS_BASE_SHA}..$GIT_HEAD"
+  echo
+  git log --no-merges --oneline "$COMMIT_RANGE"
+  echo
```

Comment:
I'm not sure if moving `git log --no-merges --oneline "$COMMIT_RANGE"` up is an improvement. At least for me, it is very convenient to look at the list of commits at the end of the log.

Reply:
Is there even any value in keeping it. Now that it only runs on PRs, it should be only be needed for rare debugging of CI infrastructure bugs? In the normal case it will just be equal to the list of commits that GitHub displays, no?

Reply:
> In the normal case it will just be equal to the list of commits that GitHub displays, no?

Exactly. That is why I check it out to ensure that it is normal :)

Anyway, I'm OK with the current state of this PR.